kbase-extension/static/kbase/js/widgets/function_output/kbaseRESKESearchResultDemo.js

```diff
@@ -0,0 +1,535 @@
+ define (
+	[
+		'kbwidget',
+		'bootstrap',
+		'jquery',
+		'plotly',
```

Comment:
It doesn't look like Plotly is being used in this widget. Should be removed as a dependency.